run_dir/design/element_flowcells.html

```diff
@@ -21,8 +21,8 @@ <h1>
         </thead>
         <tfoot>
             <tr class="darkth">
+                <th class="sort searchable" data-sort="ngi_run_id">NGI Run ID</th>
                 <th class="sort searchable" data-sort="start_date">Start date</th>
-                <th class="sort searchable" data-sort="run_name">Run name</th>
                 <th class="sort searchable" data-sort="run_type">Run type</th>
                 <th class="sort searchable" data-sort="side">Side</th>
                 <th class="sort searchable" data-sort="cycles">Cycles</th>
```

Comment:
Was not adding the `searchable` class to `Outcome` deliberate? It doesn't have the search box at present.

Reply:
Good catch. There's no search boxes anywhere neither on this page nor on the Illumina one. So maybe just remove the searchable tag?

Reply:
Ah sorry I looked at the wrong page. I'll fix the list of flowcells in another PR.